crates/bevy_ui/src/ui_node.rs

```diff
@@ -391,3 +391,48 @@ pub struct CalculatedClip {
     /// The rect of the clip
     pub clip: bevy_sprite::Rect,
 }
+
+/// The border radius of the node
+/// 
+/// This doesn't require a [`Border`] component
+#[derive(Component, Default, Copy, Clone, Debug, Reflect)]
+#[reflect(Component)]
+pub struct BorderRadius {
+    pub top_left: f32,
+    pub bottom_left: f32,
+    pub top_right: f32,
+    pub bottom_right: f32,
+}
+
+impl BorderRadius {
+    pub fn all(border_radius: f32) -> Self {
+        Self {
+            top_left: border_radius,
+            bottom_left: border_radius,
+            top_right: border_radius,
+            bottom_right: border_radius,
+        }
+    }
+
+    pub fn to_array(&self) -> [f32; 4] {
+        [
+            self.top_left,
+            self.bottom_left,
+            self.top_right,
+            self.bottom_right,
+        ]
+    }
+}
+
+/// The visual properties of the node's border
```

Comment:
This should link to `BorderRadius`, and explain what a `Border` component without a `BorderRadius` component means.